### ISSUE TITLE
Support <Tab>

### ISSUE THREAD

**Issue by Anton-4** [mdevlamynck/living-documentation-editor-poc#3]
Using the Tab key should bring you to the next Blank.

**Comment by mdevlamynck** [comment]
You mean directly into the player or just in the scenario file? I'm planning on adding other scenarios about navigation, editing existing code, etc. And also on adding variations to try different key bindings to see what feels nice to use and Tab is definitely among them ;)

**Comment by Anton-4** [comment]
Sounds cool :+1: I meant that the tab key could be used to transition between nodes.
For example now we have:
```
  - { from: '10', to: '11', key: '<Enter>' }
```
And I want to support
```
  - { from: '10', to: '11', key: '<Tab>' }
```

**Comment by mdevlamynck** [comment]
Yes, I was planning to add support for keys outside of simple characters as needed but maybe a quick pass to implement most common keys is needed.
